content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -89,29 +89,27 @@ the training process. To enable this, you will need to set up a
 Now we need to add a training stage to `dvc.yaml` including `checkpoint: true`
 in its <abbr>output</abbr>. This tells DVC which <abbr>cached</abbr> output(s)
 to use to resume the experiment later (a circular dependency). We'll do this
-with `dvc stage add`.
+with `dvc exp init`.
 
 ```dvc
-$ dvc stage add --name train \
-                --deps data/MNIST --deps train.py \
-                --params seed,lr,weight_decay \
-                --checkpoints model.pt \
-                --plots-no-cache predictions.json \
-                --plots-no-cache dvclive/scalars \
-                --metrics-no-cache dvclive.json
-                python train.py
+$ dvc exp init \
+--data 'data/MNIST' \
+--code 'train.py' \
+--live 'dvclive' \
+--model 'model.pt' \
+--type 'checkpoint' \
+python train.py
 ```
 
-The checkpoints need to be enabled in DVC at the pipeline level. The
-`-c / --checkpoint` option of the `dvc stage add` command defines the checkpoint
-file or directory. The checkpoint file, _model.pt_, is an output from one
-checkpoint that becomes a dependency for the next checkpoint, such as the model
-weights file.
+We use the `--model` and `--type` flags to define the checkpoint file or
+directory. The checkpoint file, _model.pt_, is an output from one checkpoint
+that becomes a dependency for the next checkpoint, such as the model weights
+file.
 
-The rest of the `dvc stage add` command sets up our dependencies for running the
-training code, which parameters we want to track (which are defined in the
-_params.yaml_), some configurations for our plots, showing the training metrics,
-and specifying where the logs produced by the training process will go.
+The rest of the `dvc exp init` flags set up our dependencies for running the
+training code, which parameters we want to track (by default all inside
+_params.yaml_), and configures [DVCLive](/doc/dvclive) to be
+[used alongside DVC](/doc/dvclive/dvclive-with-dvc).
```

Comment:
Since users here might be unfamiliar with DVCLive, it might be more helpful to keep it closer to the original text, like "configures where the DVCLive logs produced by the training process will go."

Reply:
Not a strong opinion but I think if users are unfamiliar with DVCLive the phrase won't change much and a link to a more detailed doc is better.

Perhaps the better is to add the phrase + link to the doc

Reply:
Yeah, definitely agree that the links are good. Sorry that wasn't clear.